code/game/supplyshuttle.dm

```diff
@@ -327,7 +327,7 @@ var/datum/controller/supply/supply_controller = new()
 		M.count_niche_stat(STATISTICS_NICHE_CRATES)
 
 	playsound(C.loc,'sound/effects/bamf.ogg', 50, 1)  //Ehh
-	var/obj/structure/droppod/supply/pod = new()
+	var/obj/structure/droppod/supply/pod = new(null, C)
```

Comment:
```suggestion
	var/obj/structure/droppod/supply/pod = new(package = C)
```

Reply:
I wish but unfortunately you can't use kwargs in atom new() due to us using the New->Initialize scheme losing them

![image](https://github.com/cmss13-devs/cmss13/assets/604624/b5907a96-44ac-402d-b860-0e80f67c22e6)
![image](https://github.com/cmss13-devs/cmss13/assets/604624/2677295b-4f90-4a99-889a-286629473aaa)